code/site/components/com_pages/view/page/tmpl/default.html.php

```diff
@@ -9,8 +9,25 @@
 
 defined('KOOWA') or die; ?>
 
+<? // Before display - content plugin event ?>
+
+<? if($page->process->plugins) : ?>
+    <?= helper('event.trigger', array(
+        'name'       => 'onContentBeforeDisplay',
+        'attributes' => array('com_pages.page', &$page->content, (object)$page->getProperties(), 0)
```

Comment:
@johanjanssens,

`$page->content` is always null since it's already a callable method. I think it's best to get the content already before triggering plugin events. Something like `<? $content = $page->content() ?>` at the beginning and using `$content` variable when it's needed instead of `$page->content()`

Reply:
Good catch, fixed.